config/deploy/prod.rb

```diff
@@ -1,6 +1,6 @@
 server 'preservation-catalog-prod-01.stanford.edu', user: 'pres', roles: %w[m2c c2m app db web]
 server 'preservation-catalog-prod-02.stanford.edu', user: 'pres', roles: %w[cv app]
-server 'preservation-catalog-prod-03.stanford.edu', user: 'pres', roles: %w[worker app resque]
+server 'preservation-catalog-prod-03.stanford.edu', user: 'pres', roles: %w[worker app resque worker]
```

Comment:
These looks redundant?  `%w[worker app resque worker]`

Am I missing something?

Reply:
No, I was missing that we weren't using the resque role for anything, even though it was there, so yeah, redundant.